clients/repo_client.go

```diff
@@ -44,6 +44,7 @@ type RepoClient interface {
 	GetCreatedAt() (time.Time, error)
 	GetDefaultBranchName() (string, error)
 	GetDefaultBranch() (*BranchRef, error)
+	GetMFARequired() (bool, error)
```

Comment:
We may want to reach a decision on https://github.com/ossf/scorecard/issues/4049

This is technically a breaking change.